ui/app/common/displaycontrols/forms/views/showObservationFormData.html

```diff
@@ -1,5 +1,6 @@
 <div class="template-container">
     <section class="block dashboard-section">
+        <button class="confirm form-print-button" ng-if="ngDialogData.showPrintOption" ng-click="printForm()">Print</button>
```

Comment:
@kavitha-sundararajan In the old Forms2 display control, the print button is not displayed.

Reply:
Fixed